CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Pinch to exit full screen web content like videos (#75)
+- "Handle" UI element above pinned tiles to help communicate swipe-to-dismiss functionality 
```

Comment:
This is unnecessary: it's encapsulated in "Changed: appearance and animations of navigation overlay", which hasn't been released to users yet.